Realm/Tests/SwiftUISyncTestHostUITests/SwiftUISyncTestHostUITests.swift

```diff
@@ -0,0 +1,198 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2021 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+import XCTest
+import RealmSwift
+
+class SwiftUISyncTestHostUITests: SwiftSyncTestCase {
+    let application = XCUIApplication()
+    static var isDataCreated = false
+    override func setUp() {
+        continueAfterFailure = false
+        super.setUp()
+        // TODO: This should be a static method
+        if !SwiftUISyncTestHostUITests.isDataCreated {
+            let user = logInUser(for: basicCredentials(withName: #function, register: true))
+            populateRealm(user: user, partitionValue: #function)
```

Comment:
`#function` isn't doing something sensible here as it's always just "setUp()". In other places we use `#function` as the partition value as an easy way to use the test name, but here if it's intending to use a single partition for every test then it should just have a sensible partition value in a constant.